src/lib/components/identity/identity-manager.svelte

```diff
@@ -83,9 +92,16 @@
     } as TableData
 
     onMount(async () => {
```

Comment:
this onMount doesnt need to be `async`

Reply:
Yes you're right, it left it there accidentally because I had some async code in there before